src/front/app/components/globals/Dropdown.js

```diff
@@ -0,0 +1,143 @@
+import React, { Component } from 'react';
```

Comment:
Not seeing how this dropdown is gonna close when clicked outside, am I missing something?